ipn/ipnlocal/c2n.go

```diff
@@ -85,7 +85,8 @@ func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
 		writeJSON(res)
 	case "/sockstats":
 		w.Header().Set("Content-Type", "text/plain")
-		b.sockstatLogger.WriteLogs(w)
+		b.sockstatLogger.Flush()
```

Comment:
Since this has a side effects it should probably be a POST-only method, similar to `/logtail/flush`